sources/europe/KantonZuerich10cm-2015-TMS.json

```diff
@@ -0,0 +1,457 @@
+{
+    "url": "http://mapproxy.osm.ch:8080/tiles/KTZUERICH2015/EPSG900913/{z}/{x}/{y}.png?origin=nw",
+    "attribution": {
+        "text": "Kanton Zürich OF 2015",
+        "required": false
+    },
+    "type": "tms",
+    "name": "Kanton Zürich 2015  10cm",
+    "country_code": "CH",
+    "best": true,
```

Comment:
Yes, that has been wrong for a longish time and was removed in OLI when I forked.